components/roadmap-grid/today-marker-toggle.tsx

```diff
@@ -1,20 +1,30 @@
+import { Button, Skeleton, Text } from '@chakra-ui/react';
 import React from 'react';
-import SvgEyeOpenIcon from '../icons/svgr/SvgEyeOpenIcon';
 import SvgEyeClosedIcon from '../icons/svgr/SvgEyeClosedIcon';
-import { Text, Button } from '@chakra-ui/react';
+import SvgEyeOpenIcon from '../icons/svgr/SvgEyeOpenIcon';
 
 import { setShowTodayMarker, useShowTodayMarker } from '../../hooks/useShowTodayMarker';
 
 
+import { useGlobalLoadingState } from '../../hooks/useGlobalLoadingState';
 import styles from './today-marker.module.css';
 
 export function TodayMarkerToggle() {
   const showTodayMarker = useShowTodayMarker();
+  const globalLoadingState = useGlobalLoadingState();
 
   return (
-    <Button className={showTodayMarker ? styles.todayMarkerToggle : styles.todayMarkerToggleDisabled} onClick={() => setShowTodayMarker(!showTodayMarker)}>
-      { showTodayMarker ? <SvgEyeOpenIcon /> : <SvgEyeClosedIcon /> }
+    <Skeleton isLoaded={!globalLoadingState.get()}>
+      <Button
+        className={showTodayMarker ? styles.todayMarkerToggle : styles.todayMarkerToggleDisabled}
+        onClick={() => setShowTodayMarker(!showTodayMarker)}
+      >{
+        showTodayMarker ?
+          <SvgEyeOpenIcon /> :
+          <SvgEyeClosedIcon />
+      }
```

Comment:
Not a huge deal here, but this is one thing I very much dislike about jsx. imo, the ideal JSX is only distinguishable from HTML because of the unique element(component) names.

I'd much rather this line remain either unbroken, or set above. 

```suggestion
      >
      { showTodayMarker ? <SvgEyeOpenIcon /> : <SvgEyeClosedIcon /> }
```

or

```
// above somewhere
const TodayToggleSvg = showTodayMarker ? <SvgEyeOpenIcon /> : <SvgEyeClosedIcon />

<Button ... > <TodayToggleSvg /> ...</Button>
```

The further from HTML our render functions get, the harder it is to read & maintain. And then they get messier in the future.